content/articles/integrated-dns-providers.markdown

```diff
@@ -0,0 +1,77 @@
+---
+title: Integrated DNS Providers
+excerpt: Link an integrated DNS provider to your DNSimple account to manage zones at other authoritative DNS providers within DNSimple.
+categories:
+- DNS
+- Integrations
+---
+
+# Integrated DNS Providers
+
+DNSimple supports the ability to view and manage zones that are deployed in other DNS providers external to DNSimple. DNSimple calls such providers "integrated DNS providers" and zones ["integrated zones"](/articles/managing-integrated-zones).
+
+## Supported Features
+
+- **Import integrated zones**: When you link an integrated DNS provider to your DNSimple account, all the zones hosted on that integrated DNS provider will be imported into DNSimple and listed on the [Zones](/articles/managing-integrated-zones) page.
+- **Management of integrated zone records**: List, create, update, and delete integrated zone records from DNSimple using the [Deployment Editor](/articles/deployment-editor). (Note: The list of supported record types differs based on the provider)
+- **2-way Syncing of Records**: Sync your zone records from the integrated zone to DNSimple, or from DNSimple to the integrated zone, with the [Deployment Editor](/articles/deployment-editor#record-syncing).
+
+## Supported Integrated DNS Providers
+
+To manage [integrated zones](/articles/managing-integrated-zones) from DNSimple, you first have to link an integrated DNS provider to your DNSimple account.
+
+DNSimple supports the management of zones from the authoritative DNS providers listed below.
+
+### Amazon Route 53
+
+DNSimple supports [Amazon Route 53](https://aws.amazon.com/route53/) as an integrated DNS provider.
+
+#### Prerequisites for linking Amazon Route 53 as an integrated DNS provider
+
+To link [Amazon Route 53](https://aws.amazon.com/route53/) as an integrated DNS provider, you need:
+
+- An [AWS access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for an AWS user with permission to manage public hosted zones at Route 53, including these permissions:
+  - route53:ListHostedZones
+  - route53:ListHostedZonesByName
+  - route53:CreateHostedZone
+  - route53:ListResourceRecordSets
+  - route53:ChangeResourceRecordSets
+  - route53:GetHostedZone
+  - route53:DeleteHostedZone
+- Administrator access to a DNSimple account
+
+## Supported record types {#supported-record-types}
+
+Records of supported record types can be [synced](/articles/deployment-editor#record-syncing) from/to DNSimple to/from an integrated DNS provider.
+However, record fields not supported by DNSimple or the destination provider will be ignored. For example, record fields, like [record notes](/articles/record-notes) and [regional record](/articles/regional-records) information, are only supported in DNSimple records. They will not be synced to the integrated DNS provider.
+
+#### Supported Route 53 record types
```

Comment:
I think we'll need to add URL here once https://github.com/dnsimple/dnsimple-app/pull/22728 is merged

Reply:
Perhaps with a note that explains a bit (eg. it is transformed into an A record).

Reply:
Yes, I'll open another PR to include the URL record type after https://github.com/dnsimple/dnsimple-app/pull/22728 is merged 👍